django-stubs/contrib/admin/options.pyi

```diff
@@ -135,7 +135,7 @@ class ModelAdmin(BaseModelAdmin):
     delete_selected_confirmation_template: str = ...
     object_history_template: str = ...
     popup_response_template: str = ...
-    actions: Sequence[Callable[[ModelAdmin, HttpRequest, QuerySet], None]] = ...
+    actions: Sequence[Union[Callable[[ModelAdmin, HttpRequest, QuerySet], None], str]] = ...
```

Comment:
I guess that this PR has unrelated changes from other PRs. Is that the case, @kszmigiel?

Reply:
Yes, I've updated stub files to the `master` version, I can revert it if needed (this commit: https://github.com/typeddjango/django-stubs/pull/427/commits/b032a3dd494e97a5e8f81bef80596a470a682874)

Reply:
The proper way to do it is to `rebase` this PR based on current `master`

Reply:
👍 It'll quite a challenge for me, but I'll try